requirements.txt

```diff
@@ -1,13 +1,18 @@
+coreschema==0.0.4
+coreapi==2.3.3
+coverage==4.5.2
+coveralls==1.5.1
 dj-database-url==0.5.0
 Django==2.1.5
 django-cors-middleware==1.3.1
 django-extensions==2.1.4
 djangorestframework==3.9.0
+drf-yasg==1.12.1
+openapi-codec==1.3.2
 psycopg2==2.7.6.1
 psycopg2-binary==2.7.6.1
 PyJWT==1.7.1
 python-decouple==3.1
 pytz==2018.9
+ruamel.yaml==0.15.85
```

Comment:
@calvinpete did you use `pip freeze` to generate this file. otherwise what does this requirement do.